get5/models.py

```diff
@@ -419,6 +419,9 @@ def add_if(key, value):
             d['maplist'] = []
             for map in self.veto_mappool.split():
                 d['maplist'].append(map)
+				
+        d['min_spectators_to_ready'] = 0
```

Comment:
I'd rather this not be added unless it actually serves some purpose - the get5 default is 0 so this doesn't seem to.

Reply:
@splewis 
I tested the feature last week on our lan-event.
I didnt encountered breaking bugs.

And you are right "min_spectators_to_ready" shouldn't be static. 
It should be also configuratable. At least as "optional".
I change this to be optional configuratable.